src/axom/sina/tests/src/sina_Run.cpp

```diff
@@ -0,0 +1,98 @@
+#include <stdexcept>
```

Comment:
This needs a copyright header

Reply:
Should I just go ahead and put a copyright header on every header and test file for Sina?

Reply:
Yes please. We have a script to help with that -- 
https://github.com/LLNL/axom/blob/develop/scripts/copyrightPrepender.py